lib/ash/actions/aggregate.ex

```diff
@@ -81,32 +81,65 @@ defmodule Ash.Actions.Aggregate do
                                     metadata do
             Ash.Tracer.set_metadata(opts[:tracer], :action, metadata)
 
+            # Preserve the original tenant from opts before handle_multitenancy might clear it
+            original_tenant = opts[:tenant] || query.tenant
```

Comment:
I believe we updated `Ash.Actions.Read.handle_multitenancy` to not clear the tenant, so we shouldn't need to do this.